otter/test/convergence/test_steps.py

```diff
@@ -469,9 +469,9 @@ def test_add_nodes_to_clb_terminal_failures(self):
         if there is any other 4xx error, then
         the error is propagated up and the result is a failure.
         """
-        terminals = (CLBDeletedError(lb_id=u"12345"),
+        terminals = (CLBNotFoundError(lb_id=u"12345"),
+                     CLBDeletedError(lb_id=u"12345"),
                      CLBNodeLimitError(lb_id=u"12345"),
-                     NoSuchCLBError(lb_id=u"12345"),
```

Comment:
you mean in the steps.py code? the whole point of this change is to make it so we don't have to enumerate all the various ways in which CLBs can disappear... but maybe I'm misunderstanding you